### PR TITLE
QueryPage: Fix "contains 1 abstract method"

### PR DESCRIPTION
**Pull request by paladox**
In MW 1.43, getQueryInfo was made abstract so we have to declare it. Before MW 1.43 it returned null so we return null here.

Also fix getQueryInfo  return type in PropertiesQueryPage.php. It has to match the one in QueryPage.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new method to retrieve query-related information.

### CODE REVIEW

File: includes/querypages/QueryPage.php [hunk 1]
```diff
@@ -247,4 +247,11 @@ function doQuery( $offset = false, $limit = false, $property = false ) {
 
 		return $num;
 	}
+
+	/**
+	 * @return array|bool
+	 */
+	public function getQueryInfo() {
+		return null;
```
Comment (JeroenDeDauw) [inline includes/querypages/QueryPage.php:255]: Does not match the type doc. If the real type is array|bool, then you can do

```php
public function getQueryInfo(): ?array {
```

Comment (paladox) [reply includes/querypages/QueryPage.php:255]: Thanks, done.

Comment (JeroenDeDauw) [reply includes/querypages/QueryPage.php:255]: Why not the PHP type instead of the comment?

Comment (paladox) [reply includes/querypages/QueryPage.php:255]: Oh, I didn’t realise you wanted that. Done now. Thanks!

File: includes/querypages/QueryPage.php [hunk 2]
```diff
@@ -247,4 +247,11 @@ function doQuery( $offset = false, $limit = false, $property = false ) {
 
 		return $num;
 	}
+
+	/**
+	 * @return array|null
+	 */
+	public function getQueryInfo(): ?array {
```
Comment (JeroenDeDauw) [inline includes/querypages/QueryPage.php:254]: Once the type is in PHP, the comment becomes noise, so is better removed

**Pull request merged**